modules/sd-planning-considerations.adoc

```diff
@@ -56,12 +56,11 @@ If you change the number of compute nodes after installation, the control plane
 
 Post-installation scaling requirements for control plane and infrastructure nodes are assessed on a case-by-case basis. Node resource consumption and received alerts are taken into consideration.
 
-.Rules for control plane node resizing alerts
+.Rules for control plane node resizing alert
 
-Resizing alerts are triggered for the control plane nodes in a cluster when either of the following scenarios are true: 
+A resizing alert is triggered for the control plane nodes in a cluster when the following occurs:
 
-* Each control plane node has less than 16GiB RAM, and there are more than 25 and less than 101 compute nodes.
-* Each control plane node has less than 32GiB RAM, and there are more than 100 compute nodes.
+* Control plane nodes sustain over 66% utilization on average in a classic ROSA cluster.
```

Comment:
This update leaves a single item on the list, unlike before when there was more than one condition to be met. Can we revise this to drop the list formatting then? For example:

> A resizing alert is triggered for the control plane nodes in a cluster when the control plane nodes sustain over 66% utilization on average in a classic ROSA cluster.

Reply:
I think the single item list is more consistent how it is because this whole section is very similar to `Rules for infrastructure node resizing alerts` below, which uses a list since it has two items.

If this control plane resizing was its own standalone section I think reformatting could make more sense though

Reply:
Sorry, but I disagree. It's the `.Block title` (_Rules for control plane node resizing alert_ and _Rules for infrastructure node resizing alerts_) that provide the required consistency. Here a list is not required, unless there is more than one item.

Reply:
I agree that this would normally require to be removed from the list formatting, but in order to keep the consistency with the other rules, and allow the alert to stand out, we'll keep it as a list and address it in the future. Thanks!